### ISSUE TITLE
Unable to reproduce the result

### ISSUE THREAD

**Issue by tcsere07** [vikas95/Pref_Suff_Span_NN#1]
Hi, thx for sharing your code. I've run your code, but the results are not consistent with the paper. The result should be 87.54, but I got 83.99.

current best score is 0.8399034593724859
- early stopping 30 epochs without improvement
2018-11-26 00:25:56.636704: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: Tesla K80, pci bus id: 0000:08:00.0, compute capability: 3.7)
Testing model over test set
INFO:tensorflow:Restoring parameters from results/crf/model.weights/
Restoring parameters from results/crf/model.weights/
- test acc 98.01 - f1 83.99

**Comment by vikas95** [comment]
Hi,

Sorry, I dint see this issue until now.
We were getting higher performance specifically with FastText Dutch embeddings. 
The hyperparameters for Dutch were different from other experiments. If required, I can rerun the Dutch experiment and share the trained model. 

Thanks,
-Vikas

**Comment by extremin** [comment]
Yes, I hope you could rerun the experiment and share the model, Thx.

**Comment by samarohith** [comment]
Even for Spanish, I am unable to reproduce the result in the paper. Are the hyperparameters different for Spanish too?

**Comment by vikas95** [comment]
Hi,

Did you run build_data.py first to generate all the required dictionaries from Spanish CoNLL data? 

We had used FastText embeddings for Spanish too. It would be helpful if you can specify the difference in performance you observed and the used hyper parameters. Also, the tensor flow version. 

Thanks.
